docs/names-and-ids.md

```diff
@@ -0,0 +1,43 @@
+# Names and IDs
+
+version .1
+
+There are a number of names and IDs used in THIEF.  This table describes some of them:
+
+| Identifier | format | Meaning |
+| - | - | - |
+| servicePool | string | Name of pool that a service app belongs to. |
+| requestedServicePool | string | Name of pool that a device app would like to pair with. |
+| runId | guid | Guid representing a running executable. |
```

Comment:
Is this device-specific?

Reply:
Yes, but the names are still confusing.  I think I'm going to collapse runId and sessionId/pairingId into a single value -- the reason I made them separate is no longer valid.    Since we need an ID for the device app and  different ID for the service app, maybe "serviceAppSessionId" and "deviceAppSessionId".  What do you think of that?

Reply:
I'm fine with that, but I'd use `serviceRunId` and `deviceRunId` for brevity, as I think it is sufficiently descriptive